test/e2e/framework/util.go

```diff
@@ -3551,15 +3551,9 @@ func IssueSSHCommandWithResult(cmd, provider string, node *v1.Node) (*SSHResult,
 
 func IssueSSHCommand(cmd, provider string, node *v1.Node) error {
 	result, err := IssueSSHCommandWithResult(cmd, provider, node)
-	if result != nil {
-		LogSSHResult(*result)
-	}
-
-	if result.Code != 0 || err != nil {
-		return fmt.Errorf("failed running %q: %v (exit code %d)",
-			cmd, err, result.Code)
+	if err != nil {
```

Comment:
The code in `IssueSSHCommandWithResult` should also be changed.
Check `err` first then check `result.Code`.